sbnanaobj/StandardRecord/classes_def.xml

```diff
@@ -6,7 +6,7 @@
 
 <lcgdict>
   <class name="caf::StandardRecord" ClassVersion="15">
-   <version ClassVersion="15" checksum="1805006132"/>
+   <version ClassVersion="15" checksum="1651571235"/>
```

Comment:
There is a change of checksum, which is Bad.
The recommendation is to restore the old line and then rebuild:
```suggestion
```
At that point, ROOT will add another checksum and bump the version, and that new `classes_def.xml` file should be committed for this PR.